fission/function.go

```diff
@@ -134,6 +138,41 @@ func getContents(filePath string) []byte {
 	return code
 }
 
+func getPodLogs(c *cli.Context) bool {
```

Comment:
This function doesn't just get logs, it prints them too -- it should be called "show pod logs" or "print pod logs" or something like that